<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Server</class>
 <widget class="QMainWindow" name="Server">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Server</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lb_statemessage">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>25</y>
      <width>351</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>State Message</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lw_clientlist">
    <property name="geometry">
     <rect>
      <x>295</x>
      <y>5</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="te_diplaylog">
    <property name="geometry">
     <rect>
      <x>295</x>
      <y>200</y>
      <width>256</width>
      <height>166</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="lw_netseleect">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>55</y>
      <width>276</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_serveropen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>166</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>SERVER OPEN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_Serverclose">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>295</y>
      <width>276</width>
      <height>66</height>
     </rect>
    </property>
    <property name="text">
     <string>DONE</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>556</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
